sw_ddsm115_driver:
  ros__parameters:
    # RS485 serial port name for ddsm motor HAT
    port_name: "/dev/serial/by-id/usb-1a86_USB_Single_Serial_58CD181323-if00"
    # array of numbers representing DDSM115 ids in followin order: front_left_wheel, front_right_wheel, rear_left_wheel, rear_right_wheel
    wheel_ids: [1, 2, 3, 4]
    # array of wheel directions
    wheel_directions: ["forward", "backward", "forward", "backward"]
